style.css

```diff
@@ -532,3 +641,253 @@ footer {
+    overflow-x: hidden;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .Navigation {
+    display: flex;
+    text-decoration: none;
+    gap: 32px;
+    justify-content: center;
+    align-items: center;
+    margin-right: 151px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: white;
+  }
+
+  a {
+    text-decoration: none;
+    color: white;
+  }
+
+  .title {
+    margin-left: 151px;
+    margin-top: 26px;
+    margin-bottom: 26px;
+  }
+
+  .main {
+    width: 100%;
+    margin-left: 143px;
+    margin-top: 100px;
+    margin-bottom: 251px;
+  }
+
+  .intro2 {
+    width: 566px;
+    font-size: 32px;
+  }
+
+  @media screen and (min-width: 1080px) {
+    .section-1 {
+      background-image: url(./img/bg\ desktop-1.png);
+      background-size: contain;
+      background-position: 100% 20%;
+      background-repeat: no-repeat;
+      height: 798px;
+    }
+  }
+
+  .icons {
+    flex-direction: column;
+    margin-bottom: 100px;
+    margin-top: -400px;
+    gap: 20px;
+    margin-left: 36px;
+  }
+
+  .main-part {
+    display: flex;
+    flex-direction: row;
+    margin-left: 90px;
+    margin-right: 143px;
+  }
+
+  .main-part > hr {
+    flex-grow: 1;
+    height: 1px;
+    align-self: center;
+    margin-top: 120px;
+  }
+
+  .recent-work {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .placeholder-multistories {
+    margin-top: 0;
+    margin-right: 0;
+    margin-left: 90px;
+    margin-bottom: 24px;
+    display: inline-block;
+  }
+
+  .multistories {
+    margin-top: 0;
+    margin-right: 143px;
+    gap: 24px;
+  }
+
+  .multistories > h2 {
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .multistories > p {
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .cards-whole {
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    grid-template-rows: 1fr 1fr;
+    margin-left: 90px;
+    width: 100%;
+    margin-bottom: 130px;
+  }
+
+  .cards-project {
+    display: none;
+  }
+
+  .cards {
+    height: 415px;
+    padding-top: 48px;
+    cursor: pointer;
+    width: 100%;
+    background-image: url(./img/project-img-resized.png);
+  }
+
+  .cards:hover {
+    background-image: url(./img/after-hover.png);
+    border: 2px solid black;
+  }
+
+  .cards:hover .data {
+    display: none;
+  }
+
+  .cards:hover .para {
+    display: none;
+  }
+
+  .cards:hover .html {
+    display: none;
+  }
+
+  .cards:hover .css {
+    display: none;
+  }
+
+  .cards:hover .javascript {
+    display: none;
+  }
+
+  .cards-project:hover {
+    background-color: #daa884;
+  }
+
+  .cards:hover .cards-project {
+    display: block;
+    margin-top: 315px;
+  }
+
+  .about-me {
+    margin-left: 142px;
+    margin-right: 732px;
+  }
+
+  .about-me > h1 {
+    font-size: 72px;
+  }
+
+  .about-me > p {
+    font-size: 20px;
+    width: 565px;
+  }
```

Comment:
- [OPTIONAL] The right image on the background image used does not fall at the right place on your design compare to how it is on the Figma Design. Kindly look into it and fix.

### Your Design
![your-about-logo](https://user-images.githubusercontent.com/26597432/220603419-29245b26-e69c-49d3-8603-85d246d83c7a.png)

### Figma Design
![temp-about-logo](https://user-images.githubusercontent.com/26597432/220603486-402b5dd1-ff09-4f4b-8b5e-aefc82ad1aba.png)